src/main/scala/cromwell/binding/values/WdlGcsObject.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.binding.values
+
+import cromwell.util.GoogleCloudStoragePath
+
+object WdlGcsObject {
+  def apply(value: String): WdlGcsObject = WdlGcsObject(GoogleCloudStoragePath(value))
+  def apply(bucket: String, objectName: String): WdlGcsObject = WdlGcsObject(GoogleCloudStoragePath(bucket, objectName))
+}
+
+import scala.util.{Success, Try}
```

Comment:
Unless you're trying to take advantage of scoping it's generally better to put all of your imports at the start